docs/workflow.md

```diff
@@ -131,6 +131,20 @@ $ kubecfg update --ignore-unknown=true --gc-tag kube_prod_runtime kubeprod-manif
 
 NOTE: There is a [bug in kubecfg](https://github.com/ksonnet/kubecfg/issues/211) that always requires using `--ignore-unknown=true`, even if this command-line flag defaults to `true`.
 
+### Upgrading
+
+To upgrade BKPR, download and extract the latest BKPR release from the [releases page](https://github.com/bitnami/kube-prod-runtime/releases).
+
+Next, edit the `kubeprod-manifest.jsonnet` file and update the `import` statement to point to the location of the manifests from the downloaded BKPR release.
+
```

Comment:
Since this is the non-quickstart, we should mention somewhere here to check release notes for any required manual upgrade steps.